src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public abstract double getArea();
+
+    String toString();
```

Comment:
this is already a method of Object class - we don't have to declare it once more anywhere